portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -89,4 +103,18 @@ private String getParameterWithDefault(HttpServletRequest request, String name,
   private static final DatastoreService DEFAULT_DATASTORE_SERVICE 
                           = DatastoreServiceFactory.getDatastoreService();
   private static final Gson GSON = new Gson();
+  private static final String DEFALUT_COMMENTS_TO_SHOW = "0";
+
+  /** Class that stores the content of a comment and it's id in the datastore */
+  private static final class Comment {
```

Comment:
could you ask virtual support for the best practice on such use cases?
I doubt if it provides good readability. (In c++, we normally put such classes right before the functions that use it)

Reply:
I found a better way to do that without using a nested class.
Instead, I passed a list of entities in the doGet, and change some code in script.js.